templates/node_pxe.conf.j2

```diff
@@ -0,0 +1,13 @@
+{% if use_kickstart %}
```

Comment:
This should probably not be a conditional, it should always be used. I think we have two cases.
1) The DHCP server handles DHCP and kickstart file management (use_kickstart = false)
2) The DHCP server handles DHCP but not kickstart file manangement (use_kickstart = true)

In 2) we still want to set the kickstart URL, but we don't want to generate the configs. Currently it won't give the kickstart parameter if use_kickstart = false.